docs/kepler-operator/installation/powermonitor-operator-openshift.md

```diff
@@ -0,0 +1,449 @@
+      - process   # Process-level power consumption
+      - container # Container-level power consumption
+```
+
+#### Timing Settings
+
+Configure sample rates and staleness thresholds:
+
+```yaml
+spec:
+  kepler:
+    config:
+      sampleRate: 5s      # How often to sample metrics
+      staleness: 500ms    # How long before values are considered stale
+```
+
+#### Security Configuration
+
+Configure security mode and service accounts:
+
+```yaml
+spec:
+  kepler:
+    deployment:
+      security:
+        mode: rbac  # Options: none, rbac
+        allowedSANames:
+        - "my-service-account"
+```
+
+#### Resource Management
+
+Control terminated workload tracking:
+
+```yaml
+spec:
+  kepler:
+    config:
+      maxTerminated: 500  # Track top 500 terminated workloads
+      # maxTerminated: 0   # Disable terminated workload tracking
+      # maxTerminated: -1  # Track unlimited terminated workloads
+```
+
+#### Additional Configuration
+
+Use additional ConfigMaps for extended configuration:
+
+```yaml
+spec:
+  kepler:
+    config:
+      additionalConfigMaps:
+      - name: my-custom-config
+      - name: extended-settings
+```
+
+### Advanced Configuration
+
+#### Node Selection and Tolerations
+
+Control where Kepler pods are scheduled:
+
+```yaml
+spec:
+  kepler:
+    deployment:
+      nodeSelector:
+        kubernetes.io/os: linux
+        node-type: worker
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+```
+
+## Monitoring and Grafana Setup
+
+### Enable User Workload Monitoring
+
+Ensure User Workload Monitoring is enabled in your cluster:
+
+```bash
+oc -n openshift-monitoring get configmap cluster-monitoring-config -o yaml
+```
+
+If not enabled, create the configuration:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+```
+
+### Access OpenShift Metrics
+
+Navigate to **Observe** → **Metrics** in the OpenShift console:
+
+![OpenShift Metrics Dashboard Overview](assets/powermonitor-openshift/08-openshift-metrics-dashboard-overview.png)
+
+OpenShift metrics dashboard showing power consumption overview
+
+View detailed power consumption metrics:
+
+![OpenShift Metrics Dashboard Detailed](assets/powermonitor-openshift/09-openshift-metrics-dashboard-detailed.png)
+
+Detailed OpenShift metrics dashboard with power consumption charts and node information
+
+### Service Monitor Configuration
+
+The operator automatically creates ServiceMonitor resources, but you can verify:
+
+```bash
+oc get servicemonitor -n kepler-system
+```
+
+### Grafana Dashboard
+
+For advanced visualization, you can import the Kepler Grafana dashboard:
+
+```bash
+# Get the dashboard JSON
+curl -O https://raw.githubusercontent.com/sustainable-computing-io/kepler-operator/main/hack/dashboard/assets/kepler/dashboard.json
+
```

Comment:
Should we in more detail explain the dashboards in more detail? We can use information from the Power Monitor Dashboard Documentation google doc.